src/types/posts.ts

```diff
@@ -115,12 +115,26 @@ export type PostsState = {
     postsInChannel: Dictionary<Array<PostOrderBlock>>;
     postsInThread: RelationOneToMany<Post, Post>;
     reactions: RelationOneToOne<Post, Dictionary<Reaction>>;
-    openGraph: RelationOneToOne<Post, OpenGraphMetadata>;
+    openGraph: RelationOneToOne<Post, Dictionary<OpenGraphMetadata>>;
```

Comment:
Why does this need to be a Dictionary type here?

Reply:
Hi @nevyangelova 

This is needed because it is how openGraph is accessed, see https://github.com/mattermost/mattermost-redux/blob/f203bb743267cc1bf958ce3cea5a34fb19dd9bfe/src/selectors/entities/posts.ts#L78

Reply:
Good catch. The redux state was set up to potentially support multiple OpenGraph previews in the future